src/my.js

```diff
@@ -552,4 +552,25 @@ exports.wait_for = function (fn, myargs, result_name, no_error) {
     }
 
     return (result);
-};
\ No newline at end of file
+};
+
+module.exports = {
+  require:    require,
+  fibers:    fibers,
```

Comment:
I don't recall ever exporting fibers. Let's try to see if there are places we can reduce the items exported and only export functions that are used outside the module.

I know you were looking at using fibers, but I'm not sure it makes the best practice for a clean node.js library.

BTW, a few years ago I attempted a pretty major re-write such that the synchronous and asynchronous logic could be combined into a single set of code. I believe that is where I started exploring using fibers. I never finished getting the re-write back up to the same level of functionality.

https://github.com/jadonk/bonescript/commit/b5a0269bbf0426240431bb44933a30179e79ab03

Reply:
This was a mistake from my side , i will correct it now .I also agree with reducing the unused exported items. I will try to find alternative ways to somehow combine synchronous and asynchronous logic to single code-set.